### DISCUSSION TITLE
Build Failed  Command "npm run build" exited with 1

### DISCUSSION THREAD

**Discussion by JamilW** [vercel/next.js#65430] [Help]
### Summary

Whenever I try to deploy my latest changes I get a "Build Failed

Command "npm run build" exited with 1" error. I suspect it has something to do with .env files, which are supposed to be untracked as I have moved them into the root directory of my repo, which is a next js 13 blog. I have tried to overide the build by changing "next build" to "CI = false" or "npm run build" and keep getting errors. Need help.

**Comment by icyJoseph** [comment]
Show more logs, or info about the build error.

**Comment by ghost** [comment]
Hi, @JamilW I am having the same issue. Did you get any solution to this error?
